proto/tikvpb.proto

```diff
@@ -146,12 +138,8 @@ message BatchCommandsRequest {
             kvrpcpb.CheckTxnStatusRequest CheckTxnStatus = 25;
             kvrpcpb.TxnHeartBeatRequest TxnHeartBeat = 26;
 
-            kvrpcpb.VerGetRequest VerGet = 27;
```

Comment:
The original `VerGetRequest` definition is still there, what is the plan to deprecate or remove?

Reply:
My mistakes. I will remove them.

Reply:
Addressed at https://github.com/pingcap/kvproto/pull/754

PTAL @andylokandy @ekexium